docs/src/route.ts

```diff
@@ -22,7 +22,7 @@ const ROUTES = {
   showcase: '/material-ui/discover-more/showcase/',
   coreRoadmap: '/material-ui/discover-more/roadmap/',
   vision: '/material-ui/discover-more/vision/',
-  support: '/material-ui/getting-started/support/#professional-support-premium',
```

Comment:
@michelengelen Changing this to just the support page unless you want it to go to https://mui.com/x/introduction/support/#technical-support

Reply:
No, just the support page is fine, since material-ui technically not covered by the X support (internally it kinda is, since we have ownership of both). We still very much prefer github issues to be opened in the respective repos, so the redirect to X would be bad in that regard